src/roots.jl

```diff
@@ -9,69 +9,6 @@ Base.size(x::Interval) = (1,)
 
 isinterior{N}(X::IntervalBox{N}, Y::IntervalBox{N}) = all(isinterior.(X, Y))
```

Comment:
This should probably go in `IntervalArithmetic`.

Reply:
Good point.